gcc/testsuite/rust/compile/shadow1.rs

```diff
@@ -2,5 +2,5 @@ fn main() {
     let mut x = 5;
     let mut x;
     x = true;
-    x = x + 2; // { dg-error "cannot apply this operator to types bool and <integer>"  }
+    x = x + 2; // { dg-error "cannot apply operator '\\+' to types bool and <integer>"  }
```

Comment:
```suggestion
    x = x + 2; // { dg-error "cannot apply operator .+. to types bool and <integer>"  }
```

Reply:
for future reference, this would work too I believe. but you don't have to change it here :)